status-of-this-document.md

```diff
@@ -1 +1,2 @@
 This is a W3C Group Note on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT). The purpose of this work is to update the previous WCAG2ICT Note's guidance to include changes made in WCAG 2.1 and 2.2.
+To comment, file an issue in the [W3C WCAG2ict](https://github.com/w3c/wcag2icct/) GitHub repository. Create separate GitHub issues for each topic, rather than commenting on multiple topics in a single issue. It is free to create a GitHub account to file issues. If filing issues in GitHub is not feasible, send email to public-agwg-comments@w3.org (comment archive).
```

Comment:
WCAG2ICT is all caps

```suggestion
To comment, file an issue in the [W3C WCAG2ICT](https://github.com/w3c/wcag2icct/) GitHub repository. Create separate GitHub issues for each topic, rather than commenting on multiple topics in a single issue. It is free to create a GitHub account to file issues. If filing issues in GitHub is not feasible, send email to public-agwg-comments@w3.org (comment archive).
```